docs/design/admission_control_limit_range.md

```diff
@@ -73,8 +73,8 @@ type LimitRangeItem struct {
   Min ResourceList `json:"min,omitempty" description:"min usage constraints on this kind by resource name"`
   // Default resource limits on this kind by resource name
   Default ResourceList `json:"default,omitempty" description:"default resource limits values on this kind by resource name if omitted"`
-  // DefaultRequests resource requests on this kind by resource name
-  DefaultRequests ResourceList `json:"defaultRequests,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
+  // DefaultRequest resource requests on this kind by resource name
```

Comment:
This comment is confusing (I see you tried to make it parallel to the one for Default but it doesn't really work). Better would be just "Default resource requests on this kind..." If there is some style rule that you have to start it with the name of the field, then I would say "DefaultRequest is default resource requests on this kind..."

Also in both this line and the Default line I would say "if [resource limits/resource requests] is omitted" to make it clear what you're talking about.